src/com/raghav/javabasics/CompareTwoNumbersReturnMax.java

```diff
@@ -0,0 +1,17 @@
+package com.raghav.javabasics;
+//Given two numbers, write a program to compare the numbers and return the max among them.
+
+public class CompareTwoNumbersReturnMax {
+    public static void main(String args[]){
+        CompareTwoNumbersReturnMax max =new CompareTwoNumbersReturnMax();
+        System.out.println(max.findMax(5, 10));
+        System.out.println(max.findMax(0,2));
```

Comment:
What about the case where num1=num2? Add test for that case as well.
And the case where you pass negative numbers.